polygeist/tools/cgeist/Options.h

```diff
@@ -13,9 +13,38 @@
 #ifndef CGEIST_OPTIONS_H_
 #define CGEIST_OPTIONS_H_
 
+#include "llvm/ADT/StringRef.h"
 #include "llvm/Support/CommandLine.h"
 #include <string>
 
+/// Class to pass options to a compilation tool.
+class OptionsPasser {
```

Comment:
The name is a bit odd IMHO, maybe something like "ArgumentList" would work better and better reflect the fact that this is more or less a container.